orthoinference/src/main/java/org/reactome/orthoinference/EventsInferrer.java

```diff
@@ -75,14 +75,15 @@ public static void inferEvents(Properties props, String species) throws Exceptio
 		setDbAdaptors(dbAdaptor);
 
 		releaseVersion = props.getProperty("releaseNumber");
-		String pathToOrthopairs = props.getProperty("pathToOrthopairs");
+		String pathToOrthopairs = props.getProperty("pathToOrthopairs") + releaseVersion + "/";
```

Comment:
For creating a path string, it's usually preferable to use a library to handle the directory separator:

https://stackoverflow.com/questions/16919501/create-a-path-from-string-in-java7